app/src/main/java/deakin/gopher/guardian/view/patient/patientdata/medicaldiagnostics/MedicalDiagnosticsFragment.java

```diff
@@ -0,0 +1,88 @@
+package deakin.gopher.guardian.view.patient.patientdata.medicaldiagnostics;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.viewpager2.widget.ViewPager2;
+import com.google.android.material.tabs.TabLayout;
+import com.google.android.material.tabs.TabLayoutMediator;
+import deakin.gopher.guardian.R;
+import deakin.gopher.guardian.adapter.MedicalDiagnosticsViewPagerAdapter;
+
+public class MedicalDiagnosticsFragment extends Fragment {
```

Comment:
This file was also recently migrated to Kotlin

Reply:
Seems the app can't build or run in this state because we have MedicalDiagnosticsFragment defined twice